packages/frontend/amp/components/elements/Timeline.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { renderToStaticMarkup } from 'react-dom/server';
+import { Expandable } from '@frontend/amp/components/Expandable';
+import { css } from 'emotion';
+import { palette } from '@guardian/pasteup/palette';
+
+const eventsWrapper = css`
+    margin-left: 8px;
+
+    li:not(:last-child) {
+        border-left: 1px solid rgba(220, 220, 220, 0.5);
+    }
+`;
+
+const eventStyle = css`
+    padding-left: 17px;
+    padding-bottom: 16px;
+`;
+
+const highlight = css`
+    background-color: ${palette.highlight.main};
+`;
+
+const eventIconStyle = css`
+    ${highlight}
+
+    :before {
+        content: '';
+        width: 16px;
+        height: 16px;
+        border-radius: 100%;
+        float: left;
+        margin-left: -25px;
+        background-color: ${palette.neutral[7]};
+    }
+`;
+
+const getHTML = (events: TimelineEvent[], description?: string): string => {
+    // TODO using ternary as && doesn't seem to work here - it prints out undefined
```

Comment:
Is this TODO still supposed to be here? (don't see a ternary)

Reply:
Oh yeah thanks. I am a bit confused here about how JSX handles && when the first argument is undefined but perhaps it is now okay? Will check/remove comment but if anyone knows what to check here let me know!

Reply:
Did you resolve this>